docs/articles/lwc/localdev.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Local Development Server (Beta)
+---
+
+The Local Development server is an SFDX CLI plugin that configures and runs a Lightning Web Components-enabled server on your computer. Now you can develop Lightning Web Component modules and see live changes without publishing your components to an org.
+
+**Note**: This feature is in beta and has been released early so we can collect your feedback. It may contain significant problems, undergo major changes, or be discontinued. If you encounter any problems, or want to request an enhancement, open a [GitHub issue](https://github.com/forcedotcom/lwc-dev-server-feedback/issues). The use of this feature is governed by the [Salesforce.com Program Agreement](https://trailblazer.me/terms?lan=en).
+
+* [Setup](#setup)
+    * [System Requirements](#system-requirements)
+    * [Installation](#installation)
+    * [Troubleshooting](#troubleshooting)
+    * [Configuration for Projects (Optional)](#configuration-for-projects)
+* [Working With Salesforce Modules](#working-with-salesforce-modules)
+    * [Supported Modules](#supported-modules)
+    * [Partially Supported Modules](#partially-supported-modules)
+    * [Unsupported Modules](#unsupported-modules)
+* [Considerations](#considerations)
+* [Common Errors](#common-errors)
+    * [Using an Unsupported Dependency](#using-an-unsupported-dependency)
+    * [Using a Nonexistent Component](#using-a-nonexistent-component)
+    * [Using a Component in a Nonexistent Namespace](#using-a-component-in-a-nonexistent-namespace)
+
+
+**Setup**
+---------
+
+## System Requirements
+- Developer Hub-enabled org
+- Most recent stable version of Safari, Chrome, Firefox, or Edge web browser 
```

Comment:
It appears that Talon has an issue with Safari and it doesn't load. @ntotten can we remove Safari from the list of supported browsers for now?

Reply:
you can remove it, but lets make a note that it will be supported

Reply:
In the doc, should I say after the list of supported browsers, "Safari is not supported in the beta release."?